packages/sqlduck/src/logger/sqlduck-default-logtape-logger.ts

```
@@ -0,0 +1,7 @@
+import { getLogger } from '@logtape/logtape';
```

ESLint

[error] Unable to resolve path to module '@logtape/logtape'.

(import-x/no-unresolved)